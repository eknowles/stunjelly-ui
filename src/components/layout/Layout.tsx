import * as React from 'react';

export interface ILayoutProps {
  children?: any;
}

export default class Layout extends React.Component<ILayoutProps, {}> {
  render(): JSX.Element {
    return (
      <div>{this.props.children}</div>
    )
  }
}
