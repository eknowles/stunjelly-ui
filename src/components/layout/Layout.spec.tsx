import * as React from 'react';
import { create } from 'react-test-renderer';

import Layout from './Layout';

describe('Layout', () => {
  it('render a div', () => {
    const snap = create(<Layout />).toJSON();
    expect(snap).toMatchSnapshot();
  });
});
