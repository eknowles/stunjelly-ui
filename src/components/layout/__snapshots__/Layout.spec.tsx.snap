// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout render a div 1`] = `<div />`;
